microsoft/testsuites/vm_extensions/mde.py

```diff
@@ -0,0 +1,104 @@
+import time
+
+from typing import Any
+from pathlib import PurePath
+
+from assertpy import assert_that
+
+from lisa import (
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+from lisa.operating_system import BSD
+from lisa.testsuite import TestResult
+from lisa.tools import Curl, MDE as mdatp
+from lisa.util import LisaException, SkippedException
+
+@TestSuiteMetadata(
+    area="vm_extension",
+    category="functional",
+    description="""
+    MDE Test Suite
+    """,
+)
+class MDE(TestSuite):
+
+    def before_case(self, log: Logger, **kwargs: Any) -> None:
+        variables = kwargs["variables"]
+        self.onboarding_script = variables.get("onboarding_script", "")
+        if not self.onboarding_script:
+            raise SkippedException("Onboarding script is not provided.")
+
+    @TestCaseMetadata(
+        description="""
+            Verify MDE installation
+        """,
+        priority=1,
+        requirement=simple_requirement(min_core_count=2,
+                                       min_memory_mb=1024,
+                                       unsupported_os=[BSD])
+    )
+    def verify_mde(self, node: Node, log: Logger, result: TestResult) -> None:
+
+        #Invoking tools first time, intalls the tool.
+        try:
+            output = node.tools[mdatp]._check_exists()
+        except LisaException as e:
+            log.error(e)
+            output = False
+
+        assert_that(output).described_as('Unable to install MDE').is_equal_to(True)
```

Comment:
Is the description (`described_as`) supposed to be opposite of what is being checked (`is_equal_to`) here and elsewhere?